AsyncDisplayKit/ASEditableTextNode.h

```diff
@@ -125,6 +113,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (CGRect)frameForTextRange:(NSRange)textRange AS_WARN_UNUSED_RESULT;
 
+#pragma mark - UITableViewCell specific passthrough properties
```

Comment:
These aren't UITableViewCell-specific, of course.  Let's skip any changes to ASEditableTextNode; as distinct from ASCellNode, these properties are of central importance to ASEditableTextNode, whereas the passthroughs on the cell are used by only ~5-10% (many apps are using collection views, and many are also using custom components).